pixi.toml

```
@@ -59,10 +59,14 @@ python = "3.11.*"
 [feature.py310.dependencies]
 python = "3.10.*"
 
+[feature.pixi-pycharm.dependencies]
+pixi-pycharm = "*"
+
 [environments]
 py313 = { features = ["py313", "tests"] }
 py312 = { features = ["py312", "tests"] }
 py311 = ["py311", "tests"]
 py310 = ["py310", "tests"]
 lint = { features = ["lint"], no-default-feature = true }
 type-checking = { features = ["type-checking"] }
+pixi-pycharm = { features = ["pixi-pycharm"], no-default-feature = true }
```

```suggestion
default = ["pixi-pycharm", "tests"]
py313 = { features = ["py313", "tests"] }
py312 = { features = ["py312", "tests"] }
py311 = ["py311", "tests"]
py310 = ["py310", "tests"]
lint = { features = ["lint"], no-default-feature = true }
type-checking = { features = ["type-checking"] }
```

i would just put it in the default feature tbh